### DISCUSSION TITLE
19,998 Terminids kill on Tibit and 239 Automatons killed on Crimsica?

### DISCUSSION THREAD

**Discussion by Wrathful-Otter** [vonbeardy/hd2galaxy-issues#30] [Q&A]
I just started to look at the statistics of a few different planets and noticed that Terminids have been killed on Automaton planets and vise-versa. Have I just not been lucky enough to encounter random bots when I drop on Terminid planets?

**Comment by vonbeardy** [comment]
Not sure but that's what the APIs say...You'll also see on Super Earth and Cyberstan that there have been some illuminate kills as well.

**Reply by Wrathful-Otter** [reply]
Oh cool, honestly, I started thinking about in game lore reasons to justify it instead of think something was wrong. Like since we are told the Terminids spread via spores, and we Helldivers going back and forth fighting both sides that we carried the spores over. It's a little harder to think of how the Automatons made it to the other side, unless they are not bound as by supply routes in the same way we are.
